mtest/testutils.cpp

```diff
@@ -161,6 +161,7 @@ bool MTest::compareFilesFromPaths(const QString& f1, const QString& f2)
       {
       QString cmd = "diff";
       QStringList args;
+      args.append("-w");  // ignore any whitespace differences
```

Comment:
Unfortunately whitespace is sometimes significant in XML (at least how MuseScore uses it) so I don't think it is safe to ignore it.


![image](https://user-images.githubusercontent.com/11011881/79938240-f83b2880-8453-11ea-8c72-6c5488be9b2d.png)



```xml
          <StaffText>
            <text>This text
spans
    multiple
lines.
</text>
            </StaffText>
```